File: 2Sums.py
def two_sums(nums, target):
    i = 0
    j = 0
    print(nums)
    for i in range(len(nums)):
        for j in range(len(nums)):
            if nums[i] + nums[j] == target:
                print([i, j])
                break


two_sums([2, 7, 10, 11], 9)

File: 2Sums.js
var twoSum = function(nums, target) {
  for (var i = 0; i <= nums.length; i++) {
    for (var j = 1; j <= nums.length; j++) {
      if (nums[j] == target - nums[i]) {
        if (i == j) {
          continue;
        }
        return [i, j];
      }
    }
  }
};

console.log(twoSum([3, 2, 4], 6));
